django_emarsys/util.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import logging
+import re
+
+from django.apps import apps
+from django.conf import settings
+
+log = logging.getLogger(__file__)
+
+
+def get_model(model_string):
+    chunks = model_string.rsplit('.', 1)
+    if len(chunks) == 2:
+        app_label, model_name = chunks
+    else:
+        app_label = None
+        model_name = model_string
+
+    return apps.get_model(app_label=app_label, model_name=model_name)
+
+
+def validate_settings():
+    validate_configuration({key: getattr(settings, key)
+                            for key in dir(settings)
+                            if hasattr(settings, key)})
```

Comment:
Surprisingly yes. This happened when I still tweaked settings in the tests, so it may be an issue with overridden settings, but I included it just to be safe.

I'll comment on it in the doc string.